data/results/multimodal_data_integration/citeseq_cmbc.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "CITE-seq Cord Blood Mononuclear Cells",
+    "name": "CITE-seq Cord Blood Mononuclear Cells (duplicate)",
```

Comment:
So this is not ideal -- in this case we have two tasks using the same dataset, they should really have the same name.

Reply:
I assumed that the JSON files are auto generated so I only made the minimal changes required to get it to work, not the way it should be before this get merged. Sorry for not making this clear from my comment.

I think the main issue is that there is a citeseq_cmbc.json as well as a citeseq_cbmc.json for the same task. Is this a typo?

Reply:
Oh now I see it! Yes one of them is a typo.